LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright 2022 U.S. Federal Government (in countries where recognized) and TrussWorks
```

Comment:
We'll have to review this license file and update it for what Truss wants to use for internal projects. According to [this ADR](https://docs.google.com/document/d/12UYIN3XfRPdKQV87_0ILa9-z6eWRBcLqtwP9fCyR6Tg/edit)🔒, we should be using a mix for this project.

Reply:
What mix do you think is most appropriate? According to my reading of that TDR, this should just be CC4.0-A

Reply:
We're introducing some custom code here and there and if we ever write new plugins for Docusaurus that don't exist in the main project, we should leverage the [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0) and the. [Creative Commons — Attribution 4.0 International — CC BY 4.0](https://creativecommons.org/licenses/by/4.0/legalcode) licenses together if we can.

Reply:
Ah, I didn't read far enough in the ADR :D Yeah, we should fix the LICENSE before we merge this like you suggested.

Reply:
hmmmm what should the license look like in this case? I'm not really familiar with all the nuances there. If that needs to be resolved before the merge (makes sense), would either of y'all be willing to pair with me on getting that right? Is there anyone else we should involve in that convo?